### PR TITLE
chore: updating guidelines

### PR DESCRIPTION
**Pull request by ohager**
Guideline update

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,30 +10,56 @@ Examples of real projects in our vision
...
+## Guideline for new projects
+
 The changes on README.md must follow the following guide line to be accepted.
 
-The project must be under a category, and if a category still don't exist, it can be created (be sure to add this new category to the index on the start of the file)
+The project must be under a category, i.e. programming language, 
+and if a category doesn't exist, it can be [created](#Guideline for new categories) 
 
-The new added line on the category must follow as show bellow:
+The new added line should respect the following format:
```
Comment (erickzanardo) [inline CONTRIBUTING.md:33]: I prefer must over should here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,30 +10,56 @@ Examples of real projects in our vision
...
 
 `ROLE_TYPE` can be one of:
  - `Author`: when the person is the original author of that project
- - `Maintainer`: when the person is part of the team of the project and has contributions on it.
+ - `Co-Author`: when the person is one of the authors of that project with multiple creators/authors
+ - `Maintainer`: when the person is part of the team of the project, or has a significant degree of responsibility of that project.
+
+## Guideline for new categories 
+
+If a category is missing feel free to add one. Categories are (as of the time of writing) alnost programming languages.
```
Comment (erickzanardo) [inline CONTRIBUTING.md:46]: Is there a typo here? Almost?

Also categories are not necessary languages, Flutter for example

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -10,30 +10,56 @@ Examples of real projects in our vision
...
+When adding, you need to add a reference in the index  in __alphabetical order__  and following this format:
+
+```
+ - [Language Name](#reference-)
```
Comment (erickzanardo) [inline CONTRIBUTING.md:50]: Iníquos go with category instead of language name

**Review by erickzanardo — COMMENTED** [review]
Great additions, left some comments, let me know what you think

**Comment by gustavobigardi** [comment]
looks good to me.

**Review by gustavobigardi — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]
LGTM

**Pull request merged**